internal/pipeline/resolver.go

```diff
@@ -1,19 +1,24 @@
 package pipeline
 
-// TODO temporary type for resolvers to return. this should be amended in future work
-type Pipeline any
+import "errors"
+
+// Pipeline is a struct containing information about a pipeline for a resolver to return
+type Pipeline struct {
+	Name string
+	Org  string
+}
```

Comment:
I'm not sure what to use here. A simple struct like this we can add fields to as necessary? Or should we use an existing struct from the go-buildkite package or something? (I think a specific dependency on that would not be ideal)